src/cropduster.js

```diff
@@ -199,29 +223,34 @@ const CD = {
 
     CD.pause(options.maxSuspension || 0, msg);
```

Comment:
Why not rename this `msg` as well? (declared on line 216 above, but github won't let me put my comment up there)

Reply:
Good point, done!